models/gc_autopet_fpr/config/default.yml

```diff
@@ -0,0 +1,32 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: AutoPET False Positive Reduction nnUNet algorithm (dicom to dicom)
+
+execute:
+- FileStructureImporter
+- TciaDicomProcessor
+- AutoPETRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  FileStructureImporter:
+    input_dir: 'input_data'
+    structures:
+      - $sid@instance/re:.*::/re:.*-GK.*::@dicom:mod=ct
```

Comment:
Per model metadata, the inputs are DICOM CT and DICOM PET, see https://github.com/MHubAI/models/pull/52/files#diff-dba8e5743cedb6dd6171269267ec35e5ae16927b3e8232afea60fc4472b40743R7.

Reply:
@fedorov is right here, please refer to my first example above where I proposed a solution for this specific case. The second example is more of a general technical possibilit, hence the explicit notion of not beeing suitable for this model (_This won't fit the described use case as we've two different modalities [...]_).

Reply:
The configuration for the FileStructureImporter was configured to work out of the box for the data structure from the [FDG-PET-CT-Lesions](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=93258287) dataset. 

This of course won't work in general for other datasets, so a general structured interface like you propose is a better alternative. I propose to change the GK into CT though, because GK was specific to the FDG-PET-CT-Lesions dataset and does not make any sense out of context.

I think indeed that the first example is the clearer solution in this specific case. The structure filter will become:

```
structures:
- $sid@instance/CT@dicom:mod=ct
- $sid/PET@dicom:mod=pet
```